### ISSUE TITLE
Support for more grid columns.

### ISSUE THREAD

**Issue by Tarka-Dhal** [liip/bootstrap-blocks-wordpress-plugin#5]
Hello, is possible to create support for more columns than 12? I'm using $grid-columns: 24; in some templates to more precision layout.

Thank you.

**Comment by tschortsch** [comment]
Hi @Tarka-Dhal. In the current solution the 12 columns are hardcoded. So sadly it's not easily possible to change it to 24. The feature itself shouldn't be too hard to implement but we would then need to add a Plugin setting page itself to adjust to value. At some point we will add that but it doesn't have a high priority for us.

**Comment by ickledev** [comment]
Now that the plugin has a settings page, any chance this could creep further up the priority list?

**Comment by tschortsch** [comment]
Good catch 😉  But we will probably start with adding some already existing options to the settings page before creating new ones. The column count also has some hard-to-solve implications on the styling in the editor.
